app/src/main/java/edu/usf/sas/pal/muser/ui/HomeActivity.java

```diff
@@ -0,0 +1,19 @@
+package edu.usf.sas.pal.muser.ui;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import com.simplecity.amp_library.R;
+
+import edu.usf.sas.pal.muser.manager.UserRegistrationManager;
+
+public class HomeActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+        new UserRegistrationManager(getApplicationContext(), this)
```

Comment:
Let's try to re-work this to remove HomeActivity. So the file permission pops up first, and the `new UserRegistrationManager(getApplicationContext(), this).registerParticipant()` is called from the permissions granted callback.

Currently there are issues with the Activity backstack and the blank HomeActivity hanging around when it shouldn't. This will be a constant hassle to deal with and create some weird UI experiences on some devices.